components/site/tests/site_i18n.rs

```diff
@@ -169,6 +169,14 @@ fn can_build_multilingual_site() {
     assert!(file_contains!(public, "fr/tags/index.html", "bonjour"));
     assert!(!file_contains!(public, "fr/tags/index.html", "hello"));
 
+    // sitemap contains per-language taxonomies
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/tags/"));
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/tags/hello/"));
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/fr/tags/"));
+    assert!(file_contains!(public, "sitemap.xml", "https://example.com/fr/tags/bonjour/"));
```

Comment:
The current version puts `https://example.com/tags/bonjour/` here (notice missing `fr`), which results in lots of 404s in the crawlers.